integreat_cms/cms/templatetags/linkcheck_filters.py

```diff
@@ -49,3 +53,18 @@ def url_anchor_icon(url: Url) -> str:
     if url.anchor_status is False:
         return '<i icon-name="x" class="text-red-500"></i>'
     return '<i icon-name="check" class="text-green-500"></i>'
+
+
+@register.simple_tag
+@mark_safe
+def url_for_current_region(target: str, request: HttpRequest, **kwargs: Any) -> str:
```

Comment:
With a more generic name, it would make sense to also move this to a more generic file imo

Reply:
Moved to `url_tags.py`